src/App.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
-
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
+import Restaurant from "./Restaurant";
+import Footer from "./Footer";
 import "./App.css";
 
 const App = () => {
+  const contactDetails = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789"
+  ];
```

Comment:
It would be better to define this variable outside of the component, it is just a constant and doesn't need to be recreated every render 😄